OIPA/api/transaction/serializers.py

```diff
@@ -117,6 +117,22 @@ def update(self, instance, validated_data):
         return update_instance
 
 
+class TransactionSectorV2Serializer(serializers.ModelSerializer):
```

Comment:
You can just call this TransactionSectorSerializer and remove the old one, or are there references to the old one?